relationship.md

```diff
@@ -0,0 +1,56 @@
+# [DRAFT] Relationship among WoT Groups
+
+This document explains how different WoT groups within W3C can/should be linked to each other.
+
+- WoT Working Group (WG): Responsible for writing normative and informative W3C Technical Reports (TRs) which are the deliverables of the WG, as well as establishing liaisons with related SDOs.
+- WoT Interest Group (IG): Responsible for doing official outreach, collecting use cases to extract requirements, and organizing plugfests.
```

Comment:
Inviting and encouraging input from non-members to get new use cases and market requirements

Reply:
A non-member should join a CG I would say. Otherwise, the ownership is not clear. One thing that I can definitely add is that the outreach activity of all groups is to encourage input to WG (use cases is an easy way in this direction).